### PR TITLE
Use master.toml to build

### PR DESCRIPTION
**Pull request by nhibberd**
@charleso @markhibberd

### CODE REVIEW

File: master.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+runner = bin/ci
```
Comment (markhibberd) [inline master.toml]: Missing quotes?

**Comment by nhibberd** [comment]
@charleso @nhibberd discuss

**Comment by nhibberd** [comment]
@charleso Building nowwwww

**Comment by nhibberd** [comment]
https://ci.ambiata.com/job/p.branches.master/13/console

File: master.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[master]
+  runner = "s3://ambiata-dist/master-ambiata/master-haskell-library-0.0.1-20151013064111-d6a3241"
+  version = 1
+
+[build.master]
```
Comment (markhibberd) [inline master.toml]: perhaps `build.head` or something that doesn't clash?

**Comment by nhibberd** [comment]
Updating jobs as part of PR
```
CI_COMMAND="master build branches" ci update-job ambiata-ci p.branches p templates/branches.xml
CI_COMMAND="master build dist" ci update-job ambiata-ci pp templates/branches.xml
```

**Comment by charleso** [comment]
:+1:

**Pull request merged**